### PR TITLE
Add checksum validation for all downloaded files

### LINKED ISSUE #14: Check hash of all dependencies installed via curl/wget
To gaurd against supply chain attacks such as the one executed against CodeCov it would be prudent to verify the hash of all scripts downloaded by curl or wget before adding them to the docker image.

https://about.codecov.io/security-update/

### PR DESCRIPTION
**Pull request by mattwynne**
Fixes #14

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -180,25 +186,33 @@ RUN bash ./download-chrome.sh
...
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -f erlang.deb
 
 USER $USER
 
+# TODO: checksum
 ## As a user install node and npm via node version-manager
-RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.2/install.sh | bash \
+WORKDIR /home/$USER
+RUN curl https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.2/install.sh -o install-nvm.sh \
```
Comment (mpkorstanje) [inline Dockerfile]: This command uses a different set of options from the others. Why?

File: Dockerfile [hunk 2]
```diff
@@ -180,25 +186,33 @@ RUN bash ./download-chrome.sh
...
     && apt-get install --assume-yes --no-install-recommends \
     esl-erlang \
     elixir \
-    && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -f erlang.deb
 
 USER $USER
 
+# TODO: checksum
```
Comment (mpkorstanje) [inline Dockerfile]: This todo seems done.

File: Dockerfile [hunk 3]
```diff
@@ -180,25 +186,33 @@ RUN bash ./download-chrome.sh
 COPY scripts/install-mono.sh .
 RUN bash ./install-mono.sh
 
+# TODO: checksum
```
Comment (mpkorstanje) [inline Dockerfile]: This todo seems done.

File: Dockerfile (hunk 1 above)
Comment (mattwynne) [reply Dockerfile]: I missed it, that's all.

**Comment by mattwynne** [comment]
Thanks Rien!

**Pull request merged**